src/Telemetry/Telemetry/Telemetry.php

```diff
@@ -347,12 +348,14 @@ protected function get_user_details( string $stellar_slug = '' ) {
 			$stellar_slug = Config::get_stellar_slug();
 		}
 
-		$user = wp_get_current_user();
+		$user     = wp_get_current_user();
+		$template = Config::get_container()->get( Opt_In_Template::class );
 
 		$args = [
 			'name'        => $user->display_name,
 			'email'       => $user->user_email,
 			'plugin_slug' => $stellar_slug,
+			'opt_in_text' => $template->get_intro( $user->display_name, $stellar_slug ),
```

Comment:
Don't we allow the plugins to filter this value? Is there a chance that we will be pushing a different value than what they approved if that is the case?